simplified-viewer-epub-readium1/src/main/java/org/nypl/simplified/viewer/epub/readium1/ReaderHTTPServerAAsync.java

```diff
@@ -229,6 +230,19 @@ public static ReaderHTTPServerType newServer(
         }
 
         if (size >= 0) {
+          /**
+           * Try to get the mime type from the package manifest.
+           */
+
+          ManifestItem manifestItem = pack.getManifestItem(relative);
```

Comment:
I noticed that some calls to Package methods are synchronized to an instance of ReaderNativeCodeReadLock. `getManifestItem` does not appear to make any calls into native code, so I did not synchronize.